lib/core/notifications/components/notification-history.component.html

```diff
@@ -1,34 +1,72 @@
 <div (keyup)="onKeyPress($event)">
-    <button mat-button [matMenuTriggerFor]="menu" class="adf-notification-history-menu_button"
-            id="adf-notification-history-open-button">
-        <mat-icon>mail_outline</mat-icon>
+    <button mat-button
+            [matMenuTriggerFor]="menu"
+            class="adf-notification-history-menu_button"
+            id="adf-notification-history-open-button"
+            (menuOpened)="onMenuOpened()">
+        <mat-icon matBadge="&#8288;"
+                  [matBadgeHidden]="!notifications.length"
+                  matBadgeColor="accent"
+                  matBadgeSize="small">notifications</mat-icon>
     </button>
-    <mat-menu #menu="matMenu" [xPosition]="menuPositionX" [yPosition]="menuPositionY"
-              [overlapTrigger]="false" id="adf-notification-history-menu" class="adf-notification-history-menu">
+    <mat-menu #menu="matMenu"
+              [xPosition]="menuPositionX"
+              [yPosition]="menuPositionY"
+              id="adf-notification-history-menu"
+              class="adf-notification-history-menu">
+
+        <div id="adf-notification-history-list"
+             class="adf-notification-history-list"
+             (click)="$event.stopPropagation()">
```

Comment:
what's the point of having that?

Reply:
any click in this component would close the menu, material angular behavior.